packages/zoe/test/unitTests/contracts/test-simpleExchange.js

```diff
@@ -196,28 +193,25 @@ test('simpleExchange with multiple sell offers', async t => {
     });
 
     const alicePayments = { Asset: aliceMoolaPurse.withdraw(moola(3)) };
-    const { seat: aliceSeat1 } = await zoe.redeem(
+    // 4: Alice adds her sell order to the exchange
+    const { outcome: aliceOutcome1P } = await zoe.offer(
       aliceInvite1,
       aliceSale1OrderProposal,
       alicePayments,
     );
 
-    // 4: Alice adds her sell order to the exchange
-    const aliceOfferResult1 = aliceSeat1.addOrder();
-
     // 5: Alice adds another sell order to the exchange
     const aliceInvite2 = await inviteIssuer.claim(publicAPI.makeInvite());
     const aliceSale2OrderProposal = harden({
       give: { Asset: moola(5) },
       want: { Price: simoleans(8) },
       exit: { onDemand: null },
     });
-    const { seat: aliceSeat2 } = await zoe.redeem(
+    const { coutcome: aliceOutcome2P } = await zoe.offer(
```

Comment:
typo

Reply:
This should fail a test

Reply:
Fixed. You explain below why it didn't fail.